src/components/flow/actions/callbrain/CallBrainForm.tsx

```diff
@@ -76,9 +118,13 @@ export class BrainForm extends React.Component<CallBrainFormProps> {
 }
 
 /* istanbul ignore next */
-const mapStateToProps = ({ flowContext: { brainInfo } }: AppState) => {
+const mapStateToProps = ({
+  flowContext: { brainInfo },
+  nodeEditor: { settings },
+}: AppState) => {
   return {
     brainInfo,
+    entry: settings.originalAction.entry,
```

Comment:
I believe that since the entry is already in the state it does not need to be mapped as a prop
The `mapStateToProps` function is used to map the store state into the component props, not the component state